common/db.go

```diff
@@ -201,6 +215,39 @@ func InitDB(dbPath string) (*DB, error) {
 	}
 	d.insertWinningTicket = stmt
 
+	// Insert block header
+	stmt, err = db.Prepare("INSERT INTO blockheaders(number, parent, hash, logs) VALUES(?, ?, ?, ?)")
+	if err != nil {
+		glog.Error("Unable to prepare insertMiniHeader", err)
+		return nil, err
+	}
+	d.insertMiniHeader = stmt
+
+	// Find the latest block header
+	stmt, err = db.Prepare("SELECT * FROM blockheaders ORDER BY number DESC LIMIT 1")
+	if err != nil {
+		glog.Error("Unable to prepare findLatestMiniHeader", err)
+		return nil, err
+	}
+	d.findLatestMiniHeader = stmt
+
+	// Find all block headers sorted by number
+	stmt, err = db.Prepare("SELECT * FROM blockheaders ORDER BY number DESC")
+	if err != nil {
+		glog.Error("Unable to prepare findAllMiniHeadersSortedByNumber", err)
+		return nil, err
+	}
+	d.findAllMiniHeadersSortedByNumber = stmt
+
+	// Delete block header
+	stmt, err = db.Prepare("DELETE FROM blockheaders WHERE hash=?")
+	if err != nil {
+		glog.Error("Unable to prepare DeleteMiniHeader", err)
```

Comment:
Should be `deleteMiniHeader` to match the style used in the other error messages?

Reply:
Fixed in ced063e